docs/docsearch-scraper-config.json

```diff
@@ -0,0 +1,63 @@
+{
+  "index_name": "dev_docs",
+  "start_urls": [
+    {
+      "url": "https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/",
```

Comment:
Since there isn't a good way to leave comments in a JSON file, the explanation:

This line tells the scraper where to actually _start._ Note how there isn't a regex that captures a bunch of possibilities like the other two `start_urls`.